bin/i18n/sync-codeorg-all

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/github'
+require_relative 'i18n_script_utils'
+
+IN_UP_BRANCH = "i18n-sync-in-up-#{Date.today.strftime("%m-%d")}"
+DOWN_OUT_BRANCH = "i18n-sync-down-out-#{Date.today.strftime("%m-%d")}"
+
+def should_i(question)
+  loop do
+    print "Should I #{question}? [Yes]/Skip/Quit: "
+    response = gets.strip.downcase
+    puts ''
+    if 'yes'.start_with?(response) # also catches blank/return ;)
+      return true
+    elsif 'skip'.start_with?(response) || 'no'.start_with?(response)
+      return false
+    elsif 'quit'.start_with?(response)
+      puts "quitting"
+      exit(-1)
+    else
+      puts "Sorry, I didn't understand that.\n\n"
+    end
+  end
+end
```

Comment:
This seems copied from the DOTD script. Probably worth moving to somewhere where it can be shared?

Reply:
It was, but it's mostly there so that I can manually verify each step actually worked before moving on to the next one. Since the ultimate goal is for this to be automated and for this gatekeeper to be removed and since the DOTD version has stuff like logging that this one does not, I don't think it's worth it to try and generalize the DOTD version for a temporary feature

Reply:
SG.